dts/bindings/gpio/ti,sn74hc595.yaml

```diff
@@ -1,23 +1,58 @@
 # Copyright (c) 2022, Matthias Freese
 # SPDX-License-Identifier: Apache-2.0
 
-description: SN74HC595 SPI based GPIO expander
+description: |
+  SN74HC595 SPI based GPIO expander
+  The SNx4HC595 is an 8-bit shift register.
 
 compatible: "ti,sn74hc595"
 
 include: [gpio-controller.yaml, spi-device.yaml]
 
+# Note: There are two clock inputs, one for shifting (SRCLK) and one for storing
+# (RCLK). On a rising edge, the shift clock puts a bit into the shift register,
+# but does not make it appear on the output. On a rising edge of the store clock,
+# all shifted in bits appear on the outputs of the shift register. One can use
+# `cs-gpios` property of the shift register's parent SPI controller to work the
+# load clock. If one assigns a GPIO in the GPIO_ACTIVE_LOW configuration, then as
+# the CS pin goes high at the end of the SPI transaction, the shift register will
+# load in all shifted bits.
```

Comment:
This will not show up in the binding documentation. Why not add it to the description?

Reply:
I wasn't aware of that. I will move it to the description!